src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -509,7 +513,14 @@ private void AddFunctionDescriptors(IEnumerable<FunctionMetadata> functionMetada
             else
             {
                 _logger.AddingDescriptorProviderForLanguage(_workerRuntime);
-                _descriptorProviders.Add(new RpcFunctionDescriptorProvider(this, _workerRuntime, ScriptOptions, _bindingProviders, _functionDispatcher, _loggerFactory, _applicationLifetime));
+
+                var workerConfig = _languageWorkerOptions.Value.WorkerConfigs?.FirstOrDefault(c => c.Description.Language.Equals(_workerRuntime, StringComparison.OrdinalIgnoreCase));
```

Comment:
nit: I think assigning an instance of `RpcWorkerConfig` if the result here is `null` would simplify the code.

Reply:
I changed it slightly below so we don't need to create a worker config just to drop it. I use an internal static field directly now.